app/Http/Controllers/RecommendationController.php

```diff
@@ -9,26 +9,34 @@
 use App\Models\User;
 use App\Models\Email;
 
-
 class RecommendationController extends \Controller
 {
     protected $settings;
 
     protected $rules = [
-    'rank_character'  => 'required',
-    'rank_additional' => 'required',
-    'essay1'          => 'required',
+      'rank_character'  => 'required',
+      'rank_additional' => 'required',
+      'essay1'          => 'required',
+    ];
+
+    protected $applicant_rules = [
+      'first_name'      => 'required',
+      'last_name'       => 'required',
+      'relationship'    => 'required',
+      'phone'           => 'required',
+      'email'           => 'required',
```

Comment:
I think we can get a [little more specific](https://laravel.com/docs/5.1/validation#available-validation-rules) with these rules, `email` can be email type, while phone could be `numeric` and other fields can be `alpha`